pcip/README.md

```diff
@@ -0,0 +1,90 @@
+# Pulsar Java Contrib Improvement Proposal (PCIP)
+
+## What is a PCIP?
+
+The PCIP is a "Pulsar Java Contrib Improvement Proposal" and it's the mechanism used to propose changes to the Apache Pulsar Java Contrib codebases.
+
+The changes might be in terms of new features, large code refactoring, changes to APIs.
```

Comment:
This repository is for collecting new features and new ideas. Will this involve API changes?

Reply:
Updated